### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by vivekraghuram**
The images need to be changed but the layout is all set I think. 
The images under events are meant to show upcoming events. 

homepagescreencap

### CODE REVIEW

File: app/assets/stylesheets/home.scss [hunk 1]
```diff
@@ -1,16 +1,121 @@
...
+    @media only screen and (max-width: $large-screen) {
+      padding-top: 2.3%;
+    }
+
+    @media only screen and (max-width: $laptop-width) {
+      padding-top: 0px;
+      padding-bottom: 6.5%;
+      position: absolute;
+      bottom: 0px;
+    }
```
Comment (SamLau95) [inline app/assets/stylesheets/home.scss]: Yay for responsiveness! If you put in `$laptop-width` in [_grid_settings.scss](https://github.com/calblueprint/calblueprint.org/blob/master/app/assets/stylesheets/vendor/base/_grid-settings.scss) then you can rewrite your media queries as

    @include media($large-screen) { ... }
    // or
    @include media($laptop-width) { ... }

which is a little nicer.

**Comment by vivekraghuram** [comment]
@SamLau95 fixed!

**Comment by chenhoward** [comment]
Noice!

**Comment by SamLau95** [comment]
Yay! Merge conflicts doe :( @vivekraghuram

**Comment by vivekraghuram** [comment]
gaaaaaaahhhhh!!!!!

**Comment by wonjun** [comment]
MERGE!??!?!!!

**Comment by SamLau95** [comment]
MERGE!!!!

**Comment by wonjun** [comment]
WOoo

**Pull request merged**